bin/bios-baseline.sh

```diff
@@ -70,7 +70,7 @@ function ilo_config() {
 }
 
 function ilo_enable_ipmitool {
-    if ilorest --nologo rawpatch $HPE_IPMI_CONF 2&>1 >/dev/null; then
+    if ilorest --nologo rawpatch $HPE_IPMI_CONF > /dev/null 2>&1; then
```

Comment:
This should be tested quickly on an HPE system.

I recall having these redirects one way, and then having to fight them a bit to get ilorest to actually work (ilorest is finicky).

Reply:
Installed the rpm on wasp:
```
ncn-m001:~/heemstra # export IPMI_PASSWORD=***** && /root/bin/bios-baseline.sh --check
The running host [ncn-m001-mgmt] will have settings applied last.
Verifying 9 iLO/BMCs (non-compute nodes) match BIOS baseline spec.
================================
Checking ncn-m002-mgmt ... up-to-spec
ipmitool usage: enabled
================================
Checking ncn-m003-mgmt ... up-to-spec
ipmitool usage: enabled
================================
Checking ncn-s001-mgmt ... up-to-spec
ipmitool usage: enabled
================================
Checking ncn-s002-mgmt ... up-to-spec
ipmitool usage: enabled
================================
Checking ncn-s003-mgmt ... up-to-spec
ipmitool usage: enabled
================================
Checking ncn-w001-mgmt ... up-to-spec
ipmitool usage: enabled
================================
Checking ncn-w002-mgmt ... up-to-spec
ipmitool usage: enabled
================================
Checking ncn-w003-mgmt ... up-to-spec
ipmitool usage: enabled
================================
Checking (self) ncn-m001-mgmt ... up-to-spec
ipmitool usage: enabled
Re-run this script with --check as the first and only argument to validate spec.
See logs for contacted nodes in /var/log/metal/
ncn-m001:~/heemstra #
```